corehq/apps/repeaters/dbaccessors.py

```diff
@@ -70,23 +70,23 @@ def get_paged_repeat_records(domain, skip, limit, repeater_id=None, state=None):
         'reduce': False,
         'limit': limit,
         'skip': skip,
+        'descending': True,
     }
 
     if repeater_id and not state:
-        kwargs['startkey'] = [domain, repeater_id]
-        kwargs['endkey'] = [domain, repeater_id, {}]
+        kwargs['endkey'] = [domain, repeater_id]
+        kwargs['startkey'] = [domain, repeater_id, {}]
```

Comment:
yes, that's helpful. I figured it was something like that, but found it difficult to search for `{}` in the couch docs on querying. (I didn't look very hard). Thanks for the detailed explanation.